### ISSUE TITLE
Accessibility issues in My Access Page

### ISSUE THREAD

**Issue by nirajCITZ** [bcgov/api-services-portal#801]
Description:  Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

Helpurl:      https://dequeuniversity.com/rules/axe/4.7/color-contrast?application=axeAPI

[
    {
        "any": [
            {
                "id": "color-contrast",
                "data": {
                    "fgColor": "#ffffff",
                    "bgColor": "#6a83a7",
                    "contrastRatio": 3.87,
                    "fontSize": "12.0pt (16px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "4.5:1"
                },
                "relatedNodes": [
                    {
                        "html": "<a class=\"chakra-link css-8oz1d\" aria-current=\"page\" data-testid=\"navbar-link-My Access\"><span class=\"css-1o39rat\">My Access</span></a>",
                        "target": [
                            "a[aria-current=\"page\"]"
                        ]
                    },
                    {
                        "html": "<nav role=\"banner\" class=\"css-f4yi47\">",
                        "target": [
                            "nav"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 3.87 (foreground color: #ffffff, background color: #6a83a7, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<span class=\"css-1o39rat\">My Access</span>",
        "target": [
            "a[aria-current=\"page\"] > .css-1o39rat"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.87 (foreground color: #ffffff, background color: #6a83a7, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
    },
    {
        "any": [
            {
                "id": "color-contrast",
                "data": {
                    "fgColor": "#91949a",
                    "bgColor": "#e8e8e8",
                    "contrastRatio": 2.48,
                    "fontSize": "12.0pt (16px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "4.5:1"
                },
                "relatedNodes": [
                    {
                        "html": "<section class=\"chakra-modal__content css-1xa46hq\" role=\"dialog\" id=\"chakra-modal-24\" tabindex=\"-1\" aria-modal=\"true\" aria-labelledby=\"chakra-modal--header-24\" aria-describedby=\"chakra-modal--body-24\" style=\"opacity: 0.457224; transform: scale(0.972861) translateZ(0px);\">",
                        "target": [
                            "#chakra-modal-24"
                        ]
                    },
                    {
                        "html": "<div class=\"chakra-modal__overlay css-hdd9l7\" style=\"opacity: 0.239159;\"></div>",
                        "target": [
                            ".chakra-modal__overlay"
                        ]
                    },
                    {
                        "html": "<body class=\"chakra-ui-light\">",
                        "target": [
                            "body"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 2.48 (foreground color: #91949a, background color: #e8e8e8, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<h2 class=\"chakra-heading css-2ab7wd\">",
        "target": [
            ".css-2ab7wd"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 2.48 (foreground color: #91949a, background color: #e8e8e8, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
    },
    {
        "any": [
            {
                "id": "color-contrast",
                "data": {
                    "fgColor": "#b1b1b1",
                    "bgColor": "#e8e8e8",
                    "contrastRatio": 1.75,
                    "fontSize": "12.0pt (16px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "4.5:1"
                },
                "relatedNodes": [
                    {
                        "html": "<section class=\"chakra-modal__content css-1xa46hq\" role=\"dialog\" id=\"chakra-modal-24\" tabindex=\"-1\" aria-modal=\"true\" aria-labelledby=\"chakra-modal--header-24\" aria-describedby=\"chakra-modal--body-24\" style=\"opacity: 0.457224; transform: scale(0.972861) translateZ(0px);\">",
                        "target": [
                            "#chakra-modal-24"
                        ]
                    },
                    {
                        "html": "<div class=\"chakra-modal__overlay css-hdd9l7\" style=\"opacity: 0.239159;\"></div>",
                        "target": [
                            ".chakra-modal__overlay"
                        ]
                    },
                    {
                        "html": "<body class=\"chakra-ui-light\">",
                        "target": [
                            "body"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 1.75 (foreground color: #b1b1b1, background color: #e8e8e8, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<p class=\"chakra-text css-gr9j05\">By clicking Generate Secrets we will generate your credentials once.</p>",
        "target": [
            ".css-gr9j05"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 1.75 (foreground color: #b1b1b1, background color: #e8e8e8, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
    },
    {
        "any": [
            {
                "id": "color-contrast",
                "data": {
                    "fgColor": "#bfccd8",
                    "bgColor": "#7b92a9",
                    "contrastRatio": 1.96,
                    "fontSize": "12.0pt (16px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "4.5:1"
                },
                "relatedNodes": [
                    {
                        "html": "<button type=\"button\" class=\"chakra-button css-15qa8dy\" data-testid=\"generate-secrets-button\">Generate Secrets</button>",
                        "target": [
                            "button[data-testid=\"generate-secrets-button\"]"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 1.96 (foreground color: #bfccd8, background color: #7b92a9, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<button type=\"button\" class=\"chakra-button css-15qa8dy\" data-testid=\"generate-secrets-button\">Generate Secrets</button>",
        "target": [
            "button[data-testid=\"generate-secrets-button\"]"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 1.96 (foreground color: #bfccd8, background color: #7b92a9, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
    },
    {
        "any": [
            {
                "id": "color-contrast",
                "data": {
                    "fgColor": "#bbc8d4",
                    "bgColor": "#748ba2",
                    "contrastRatio": 2.06,
                    "fontSize": "12.0pt (16px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "4.5:1"
                },
                "relatedNodes": [
                    {
                        "html": "<button type=\"button\" class=\"chakra-button css-15qa8dy\" data-testid=\"generate-credentials-done-button\">Done</button>",
                        "target": [
                            ".chakra-modal__footer > .css-15qa8dy.chakra-button[type=\"button\"]"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 2.06 (foreground color: #bbc8d4, background color: #748ba2, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<button type=\"button\" class=\"chakra-button css-15qa8dy\" data-testid=\"generate-credentials-done-button\">Done</button>",
        "target": [
            ".chakra-modal__footer > .css-15qa8dy.chakra-button[type=\"button\"]"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 2.06 (foreground color: #bbc8d4, background color: #748ba2, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
    }
]

![image.png](https://images.zenhubusercontent.com/61a121c4a7a3264dfdc530ef/f33df759-6eef-4f0d-89d2-a0439f5e0681)
